alfresco-transformer-base/src/test/java/org/alfresco/transformer/AbstractTransformerControllerTest.java

```diff
@@ -162,17 +162,26 @@ void generateTargetFileFromResourceFile(String actualTargetExtension, File testF
 
     protected File getTestFile(String testFilename, boolean required) throws IOException
     {
+        if (System.getProperty("os.name").substring(0, 3).equals("Win"))
+        {
             File testFileUrl = null;
             try {
```

Comment:
Curly braces should be on new line and each part of try-catch should be on a new line : https://hub.alfresco.com/t5/alfresco-content-services-hub/coding-standards-for-alfresco-content-services/ba-p/290457